src/app/+admin/admin-search-page/admin-search-page.component.html

```diff
@@ -1 +1 @@
-<ds-configuration-search-page configuration="administrativeView" [context]="context"></ds-configuration-search-page>
+<ds-configuration-search-page configuration="admin" [context]="context"></ds-configuration-search-page>
```

Comment:
I don't think we can just change that here. While it may affect the keys, it also affects the configuration that gets sent to the rest api with each search. You'll notice that with this change, the private and withdrawn facets have disappeared on the admin search page.

We could change the configuration name to admin on the rest side I suppose, but that may be overkill

Reply:
@artlowel : ah, good catch & my mistake. I'll just undo this change and we can cleanup this oddly named i18n key at a later time.

Reply:
@artlowel : I've now solved this by undoing those changes & force pushing.  I just moved the existing `administrativeView.search.results.head` key up to the section of keys that it relates to.  If you could give this a quick re-review, I'd appreciate it!